app/Main.hs

```diff
@@ -210,12 +210,12 @@ whenJust item action = maybe (pure mempty) action item
 
 invokeEditor :: AppState -> Next (VgrepT AppState m Redraw)
 invokeEditor state = case views (results . currentFileName) (fmap T.unpack) state of
-    Just file -> Interrupt $ Suspend $ \environment -> do
+    Just selectedFile -> Interrupt $ Suspend $ \environment -> do
```

Comment:
Does not do any harm, some people even prefer `f $ g $ h $ a` to `f . g . h $ a`.